netZooPy/panda/panda.py

```diff
@@ -507,7 +512,7 @@ def gupdate_diagonal(diagonal_matrix, num, alpha, step):
             tfs = np.tile(self.unique_tfs, (len(self.gene_names), 1)).flatten()
             genes = np.repeat(self.gene_names,self.num_tfs)
             motif = self.motif_matrix_unnormalized.flatten(order='F')
-            force = motif_matrix.flatten(order='F')
+            force = self.motif_matrix.flatten(order='F')
```

Comment:
@dcolinmorgan why did you add `self` here?

Reply:
FUCK